src/main/java/info/ata4/bsplib/struct/DStaticPropV13.java

```diff
@@ -0,0 +1,31 @@
+package info.ata4.bsplib.struct;
+
+import info.ata4.bsplib.vector.Vector3f;
+import info.ata4.io.DataReader;
+import info.ata4.io.DataWriter;
+
+import java.io.IOException;
+
+// Strata added non-uniform scaling, replacing the uniformScale member
+public class DStaticPropV13 extends DStaticPropV10CSGO {
+
+    public Vector3f scale;
+
+    @Override
+    public int getSize() {
+        return super.getSize() + 12; // 88
+    }
+
+    @Override
+    public void read(DataReader in) throws IOException
+    {
+        super.read(in);
+        scale = Vector3f.read(in);
```

Comment:
I'm assuming reading pushes forward an offset here. If the super class has 1 float for scale, and this class has 3 floats for scale, does this correctly read and write two more floats? Would we want to just read two floats here?
I'm not super familiar with bspsrc, but thought I'd give it a review :P

Reply:
I actually misspoke earlier, because I extended the v10CSGO class for v13, which means that the super class doesn't have the uniformScale float. At the end of the day, this does only read 2 extra floats compared to v11CSGO